### DISCUSSION TITLE
NO WIDGEt table found for module androidmodule

### DISCUSSION THREAD

**Discussion by Majestychen** [AndroidIDEOfficial/AndroidIDE#1328] [General]
Please tell me how to solve this problem when previewing the new default project and displaying "no widget table found for module androidmodule::APP" on the preview page. 
Thank you!

LogReceiverImpl           I   Starting log sender handler.. 
MultiLogSenderHandler     I   Starting log receiver server socket at port 33301 
GradleBuildService        I   Showing notification to user... 
EditorActivity            I   Connected to Gradle build service 
ToolingServerRunner       I   Starting tooling API server... 
ToolingApiServer          D   ToolingApiMain             Server started. Will run until shutdown message is received...  
CachingProject            W   No tasks found in project '' 
CachingProject            D   Empty module data returned by tooling server. Ignoring... 
EditorActivity            D   Is project initialized: false 
EditorActivity            D   Sending init request to tooling server.. 
ToolingApiServer          I   ToolingApiMain             Checking gradle wrapper availability...  
ToolingApiServer          I   ToolingApiMain             Gradle wrapper is available  
ToolingApiServer          D   ToolingApiServerImpl       Got initialize request  InitializeProjectMessage(directory=/storage/emulated/0/AndroidIDEProjects/My Application, gradleInstallation=)  
ToolingApiErrorStream     E   SLF4J: No SLF4J providers were found. 
ToolingApiErrorStream     E   SLF4J: Defaulting to no-operation (NOP) logger implementation 
ToolingApiErrorStream     E   SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details. 
ToolingApiServer          I   ToolingApiServerImpl       Using Gradle wrapper for build...  
ToolingApiServer          D   StopWatch                  Connector created in 138ms  
ToolingApiServer          D   StopWatch                  Project connection established in 94ms  
ToolingApiServer          D   ToolingApiMain             Arguments from tooling client:  [--init-script, /data/data/com.itsaky.androidide/files/home/.androidide/init/init.gradle, -Pandroid.aapt2FromMavenOverride=/data/data/com.itsaky.androidide/files/home/.androidide/aapt2]  
ToolingApiServer          W   ProjectReader              Starting build. See build output for more details...  
EditorActivity            D   [onPause]  No opened files.  Opened files cache reset to null. 
ToolingApiServer          D   StopWatch                  Project read successful in 119672ms  
ToolingApiServer          D   StopWatch                  Connection to project completed in 119931ms  
AndroidModule             I   Indexing sources and classpaths for project:  :app 
StopWatch                 D   Indexing sources completed in 25ms 
AndroidModule             D   Found 1 source files. 
StopWatch                 D   Indexing classpaths completed in 8035ms 
AndroidModule             D   Found 2301 classpaths. 
StopWatch                 D   Indexing 0 bootclasspaths completed in 0ms 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
ResourceTableRegistry     I   Creating resource table for 1 resource directories 
StopWatch                 D   Indexing sources completed in 8ms 
AndroidModule             D   Found 1 source files. 
ILanguageServerRegistry   D   Dispatching ProjectInitializedEvent to language servers... 
SourceFileManager         I   Creating source file manager instance for module:  AndroidModule: :app 
BootClassProvider         D   Indexing boot classpath:  /data/data/com.itsaky.androidide/files/home/.androidide/android.jar 
EditorActivity            I   Opening file at index:  0  file:   /storage/emulated/0/AndroidIDEProjects/My Application/app/src/main/res/layout/activity_main.xml 
LanguageSpecProvider      W   Scheme file 'locals' for type 'xml' not found 
CodeEditorView            I   activity_main.xml 
CodeEditorView            I   File was not modified. Skipping save operation. 
EditorActivity            D   [onPause]  Opened files cache reset to OpenedFilesCache(selectedFile=/storage/emulated/0/AndroidIDEProjects/My Application/app/src/main/res/layout/activity_main.xml, allFiles=[OpenedFile(filePath=/storage/emulated/0/AndroidIDEProjects/My Application/app/src/main/res/layout/activity_main.xml, selection=Range(start=Position(line=0, column=0, index=-1), end=Position(line=0, column=0, index=-1)))]) 
DesignerWorkspaceFragment E  
DesignerWorkspaceFragment E  java.lang.IllegalStateException: No widget table found for module AndroidModule: :app
DesignerWorkspaceFragment E  	at com.itsaky.androidide.inflater.internal.LayoutInflaterImpl.doInflate(Unknown Source:253)
DesignerWorkspaceFragment E  	at com.itsaky.androidide.inflater.internal.LayoutInflaterImpl.inflate(Unknown Source:80)
DesignerWorkspaceFragment E  	at com.itsaky.androidide.uidesigner.fragments.DesignerWorkspaceFragment.onViewCreated(Unknown Source:116)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentStateManager.createView(Unknown Source:253)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(Unknown Source:145)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentManagerImpl.moveToState(Unknown Source:62)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(Unknown Source:35)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentActivity.onStart(Unknown Source:34)
DesignerWorkspaceFragment E  	at androidx.appcompat.app.AppCompatActivity.onStart(Unknown Source:0)
DesignerWorkspaceFragment E  	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1447)
DesignerWorkspaceFragment E  	at android.app.Activity.performStart(Activity.java:8237)
DesignerWorkspaceFragment E  	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:4089)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:235)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:215)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:187)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:105)
DesignerWorkspaceFragment E  	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2618)
DesignerWorkspaceFragment E  	at android.os.Handler.dispatchMessage(Handler.java:110)
DesignerWorkspaceFragment E  	at android.os.Looper.loop(Looper.java:219)
DesignerWorkspaceFragment E  	at android.app.ActivityThread.main(ActivityThread.java:8673)
DesignerWorkspaceFragment E  	at java.lang.reflect.Method.invoke(Native Method)
DesignerWorkspaceFragment E  	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
DesignerWorkspaceFragment E  	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1109)
DesignerWorkspaceFragment E  
DesignerWorkspaceFragment E  
UIDesignerActivity        E   Failed to generate XML code because 'Inflation failed, layout has errors.' 
BootClassProvider         D   Indexing boot classpath:  /data/data/com.itsaky.androidide/files/home/.androidide/android.jar 
EditorActivity            W   UI Designer returned invalid result  0  null 
CodeEditorView            I   activity_main.xml 
CodeEditorView            I   File was not modified. Skipping save operation. 
EditorActivity            D   [onPause]  Opened files cache reset to OpenedFilesCache(selectedFile=/storage/emulated/0/AndroidIDEProjects/My Application/app/src/main/res/layout/activity_main.xml, allFiles=[OpenedFile(filePath=/storage/emulated/0/AndroidIDEProjects/My Application/app/src/main/res/layout/activity_main.xml, selection=Range(start=Position(line=0, column=0, index=-1), end=Position(line=0, column=0, index=-1)))]) 
DesignerWorkspaceFragment E  
DesignerWorkspaceFragment E  java.lang.IllegalStateException: No widget table found for module AndroidModule: :app
DesignerWorkspaceFragment E  	at com.itsaky.androidide.inflater.internal.LayoutInflaterImpl.doInflate(Unknown Source:253)
DesignerWorkspaceFragment E  	at com.itsaky.androidide.inflater.internal.LayoutInflaterImpl.inflate(Unknown Source:80)
DesignerWorkspaceFragment E  	at com.itsaky.androidide.uidesigner.fragments.DesignerWorkspaceFragment.onViewCreated(Unknown Source:116)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentStateManager.createView(Unknown Source:253)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(Unknown Source:145)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentManagerImpl.moveToState(Unknown Source:62)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(Unknown Source:35)
DesignerWorkspaceFragment E  	at androidx.fragment.app.FragmentActivity.onStart(Unknown Source:34)
DesignerWorkspaceFragment E  	at androidx.appcompat.app.AppCompatActivity.onStart(Unknown Source:0)
DesignerWorkspaceFragment E  	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1447)
DesignerWorkspaceFragment E  	at android.app.Activity.performStart(Activity.java:8237)
DesignerWorkspaceFragment E  	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:4089)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:235)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:215)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:187)
DesignerWorkspaceFragment E  	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:105)
DesignerWorkspaceFragment E  	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2618)
DesignerWorkspaceFragment E  	at android.os.Handler.dispatchMessage(Handler.java:110)
DesignerWorkspaceFragment E  	at android.os.Looper.loop(Looper.java:219)
DesignerWorkspaceFragment E  	at android.app.ActivityThread.main(ActivityThread.java:8673)
DesignerWorkspaceFragment E  	at java.lang.reflect.Method.invoke(Native Method)
DesignerWorkspaceFragment E  	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
DesignerWorkspaceFragment E  	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1109)
DesignerWorkspaceFragment E  
DesignerWorkspaceFragment E  
UIDesignerActivity        E   Failed to generate XML code because 'Inflation failed, layout has errors.' 
EditorActivity            W   UI Designer returned invalid result  0  null 
BootClassProvider         D   Indexing boot classpath:  /data/data/com.itsaky.androidide/files/home/.androidide/android.jar 
EditorActivity            D   [onPause]  Opened files cache reset to OpenedFilesCache(selectedFile=/storage/emulated/0/AndroidIDEProjects/My Application/app/src/main/res/layout/activity_main.xml, allFiles=[OpenedFile(filePath=/storage/emulated/0/AndroidIDEProjects/My Application/app/src/main/res/layout/activity_main.xml, selection=Range(start=Position(line=0, column=0, index=-1), end=Position(line=0, column=0, index=-1)))]) 
BootClassProvider         D   Indexing boot classpath:  /data/data/com.itsaky.androidide/files/home/.androidide/android.jar

**Comment by itsaky** [comment]
This was already reported in #1327.
